Updating the Window Controls Overlay docs (#2149)
MicrosoftDocs/edge-developer: microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay.md

|:--- |:---
| `titlebar-area-x` | Distance, in `px`, of the area normally occupied by the title bar from the left side of the window |
| `titlebar-area-y` | Distance, in `px`, of the area normally occupied by the title bar from the top side of the window |
| `titlebar-area-width` | Width of the title bar area, in `px` |
| `titlebar-area-height` | Height of the title bar area, in `px` |

You can use these environment variables to position and size your own content where the title bar would normally appear, when the window controls overlay feature is disabled:

```css
#title-bar {
  position: fixed;
  left: env(titlebar-area-x, 0);
  top: env(titlebar-area-y, 0);
  height: env(titlebar-area-height, 50px);
  width: env(titlebar-area-width, 100%);
}
```

Using `position: fixed;` makes sure your title bar does not scroll with the rest of the content and instead stays aligned with the window controls overlay.

Knowing where the overlay is and how big it is is important.  The overlay might not always be on the same side of the window; on macOS, the overlay is on the left side, but on Windows, the overlay is on the right side.  Also, the overlay might not always be the same size.

> Note that the `env()` CSS function takes a second parameter useful to define the position of your app content when the window controls overlay feature is missing or disabled.

Reviewer: ```suggestion
The `env()` CSS function takes a second parameter that's useful for defining the position of your app content when the window controls overlay feature is missing or disabled.
```
